### ISSUE TITLE
Children elements of an AccessibilityElement are not visible for Appium

### ISSUE THREAD

**Issue by pespinel** [appium/appium#15108]
## The problem

Hi, recently our app has changed the default behaviour in terms of accessibility. Instead of set accessibility labels in the final elements now the labels are placed in the parent element (Which is reasonable for accessibility terms). The problem here is that Appium cannot inspect any children below an AccessibilityElement.

## Environment

* Appium version (or git revision) that exhibits the issue: 1.20.2, 1.19.1
* Last Appium version that did not exhibit the issue (if applicable): N/A
* Desktop OS/version used to run Appium: macOS Catalina
* Node.js version (unless using Appium.app|exe): v10.20.0
* Npm or Yarn package manager: npm 6.14.9
* Mobile platform/version under test: iOS
* Real device or emulator/simulator: Real devices and simulators
* Appium CLI or Appium.app|exe: cli

## Details

Here it's a silly example:

![AccessbilityElement](https://user-images.githubusercontent.com/37797002/107747423-af709b00-6d17-11eb-939d-808bc9f342d8.JPG)

This is the tree:

- Card: Parent element (AccessibilityElement)
   - Image: Children element (Image view with accessibility ID)
   - Title: Children element (Text view with accessibility ID)
   - Body: Children element (Text view with accessibility ID)

## Link to Appium logs

https://gist.github.com/pespinel/2df5b63b286fae50059fa0abd921d9b2

**Comment by pespinel** [comment]
I've just found this issue which it's related: https://github.com/appium/appium/issues/12001 It was closed ... but maybe there is a possible fix now?

**Comment by mykola-mokhnach** [comment]
Unfortunately there is no way to change anything there from our side. Please report the issue to Apple.

**Comment by pespinel** [comment]
@mykola-mokhnach I know that this is directly related to XCUITest but something can be done in the Appium side. Other frameworks (Calabash for example) retrieve correctly all the children elements below an AccessibilityElement.

Accessibility is becoming an important point in all the apps and i think that you should dig for a solution here ...
